fabric-registry-sync-v0/src/main/java/net/fabricmc/fabric/impl/registry/sync/RegistrySyncManager.java

```diff
@@ -53,6 +56,7 @@
 	static final Identifier ID = new Identifier("fabric", "registry/sync");
 	private static final Logger LOGGER = LogManager.getLogger("FabricRegistrySync");
 	private static final boolean DEBUG_WRITE_REGISTRY_DATA = System.getProperty("fabric.registry.debug.writeContentsAsCsv", "false").equalsIgnoreCase("true");
+	private static final Map<Registry<?>, Set<Identifier>> SERVER_ONLY_REGISTRY_IDS = new HashMap<>();
```

Comment:
Not a pro on registry stuff I am, but does this only work with non-dynamic registries?

Reply:
Dynamic registries do not need this as the client will always have enough information to recreate the server's registry